examples/sites/demos/pc/webdoc/import-components.md

```
@@ -14,121 +14,67 @@ npm i @opentiny/unplugin-tiny-vue -D
+    Components({
+      resolvers: [TinyVueSingleResolver]
+    }),
+    AutoImport({
+      resolvers: [TinyVueSingleResolver]
+    })
+  ]
+}
 ```
 
-Webpack
+#### 温馨提示
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)